.circleci/config.yml

```diff
@@ -7,18 +7,6 @@ workflows:
   version: 2
   test:
     jobs:
-      # Even though LaunchDarkly.TestHelpers only has two build targets-- .NET Standard 2.0
-      # and .NET Framework 4.5.2-- we run the tests in a variety of runtime platforms. That's
-      # because there can be subtle differences in how APIs like System.Net.HttpListener
-      # behave across platforms, so we want to make sure these tools work consistently in all
-      # of them. The oldest one, .NET Core 2.0, is no longer a supported platform in current
-      # LaunchDarkly tools; we still test it because in some of our projects, .NET Core 2.0
-      # might be the only way to specifically test the .NET Standard 2.0 implementation of a
-      # package (e.g. if there is also a different build target for .NET Core 2.1).
-      - test-netcore:
-          name: .NET Core 2.0
-          docker-image: microsoft/dotnet:2.0-sdk-jessie
-          test-target-framework: netcoreapp2.0
```

Comment:
Unrelated: I removed this test job because `microsoft/dotnet:2.0-sdk-jessie` no longer exists; I don't think Microsoft is offering a Docker image for .NET Core 2.0 any more. In any case, the issue described in the comment here regarding .NET Standard 2.0 testing was never relevant to this project, since it does not have a specific build target for .NET Core 2.1 and therefore testing in .NET Core 2.1 is adequate to test the .NET Standard 2.0 target.